settings.gradle.kts

```diff
@@ -13,6 +13,6 @@ dependencyResolutionManagement {
     }
 }
 
-rootProject.name = "polyfit"
+rootProject.name = "Polyfit"
```

Comment:
This used to cause errors, at least still does on my end

Reply:
Yes, the PR is still a draft so I haven't prepared it for review